test/framework/containers.py

```diff
@@ -120,6 +130,7 @@ def test_end2end_singularity_recipe(self):
         args = [
             toy_ec,
             '--containerize',
+            '--container-type=singularity',
```

Comment:
This shouldn't be needed (since default type is Singularity)?

Reply:
I thought that will be explicit and clearer.

Reply:
Sure, but this way you're not testing what the default container type is, since it's always specified.

Leaving it unspecified is intentional here, it serves as a check that Singularity is the default.